src/workflows-app/RunDetails.ts

```diff
@@ -0,0 +1,472 @@
+          _.isNil(updateWorkflowPath) && setWorkflowMetadata(metadata);
+
+          if (!_.isEmpty(metadata?.calls)) {
+            setCallObjects(metadata.calls);
+          }
+          if (!_.isEmpty(metadata?.calls) && !_.isEmpty(failedTasks)) {
+            setFailedTasks((Object.values(failedTasks)[0] as { calls: any })?.calls || {});
+            metadata.calls ? setCallObjects(metadata.calls) : setCallObjects(undefined);
+            if (
+              !_.isNil(metadata?.status) &&
+              _.includes(collapseStatus(metadata?.status), [statusType.running, statusType.submitted])
+            ) {
+              stateRefreshTimer.current = setTimeout(() => {
+                fetchWorkflowMetadata(workflowId, updateWorkflowPath);
+              }, 60000);
+            }
+          }
+          const { workflowName } = metadata;
+          !_.isNil(updateWorkflowPath) && updateWorkflowPath(workflowId, workflowName);
+        }
+      } catch (error) {
+        setLoadWorkflowFailed(true);
+        notify('error', 'Error loading run details', {
+          detail: error instanceof Response ? await error.text() : error,
+        });
+      }
+    },
+    [signal, stateRefreshTimer, cromwellProxyState]
+  );
+
+  // Fetch the workflow & calls metadata once the Cromwell URL is available.
+  useEffect(() => {
+    if (!cromwellProxyState || !cromwellProxyState.state || !workflowId) {
+      return;
+    }
+    fetchWorkflowMetadata(workflowId);
+  }, [workflowId, cromwellProxyState, fetchWorkflowMetadata]);
+
+  // Fetch the cost data for the workflow and all of its subworkflows.
+  useEffect(() => {
+    if (!cromwellProxyState || !cromwellProxyState.state || !workflowId) {
+      return;
+    }
+    const makeCostMetadataRequest = async () => {
+      const costMetadata = await fetchCostMetadata({ cromwellProxyUrl: cromwellProxyState.state, signal, workflowId });
+      setCostMetadata(costMetadata);
+      setRootWorkflowCost(calculateCostOfCallsArray(costMetadata.calls));
+    };
+    makeCostMetadataRequest();
+    setIsCostMetadataLoading(false);
+  }, [cromwellProxyState, signal, workflowId]);
+
+  // Given a call name (e.g. main_worfklow.taskName or main_workflow.subworkflowName, return the cost of that call, or undefined if it can't be found.
+  // Passed to child components so they can calculate the cost of individual calls using the cost metadata this page fetched.
+  const getCostOfCallFn = useCallback(
+    (callName: string): number | undefined => {
+      if (!costMetadata || !costMetadata.calls) {
+        return undefined;
+      }
+      const foundCallAttempts = findCallAttemptsByCallNameInCostGraph(callName, costMetadata);
+      if (!foundCallAttempts) {
+        console.error('Call not found', callName, costMetadata.calls);
+        return undefined;
+      }
+      return sumCostsOfCallAttempts(foundCallAttempts);
```

Comment:
As we discussed earlier, I believe each attempt appears in its own row in the CallTable, if so they should not be summed.